admin_class.php

```diff
@@ -1,690 +1,1153 @@
+function login(){
```

Comment:
I don't see the initial lines fo the session_start() added to this file - Please cross check and add it:
```
<?php
session_start();
ini_set('display_errors', 1);
Class Action {
	private $db;

	public function __construct() {
		ob_start();
   	include 'db_connect.php';
    
    $this->db = $conn;
	}
	function __destruct() {
	    $this->db->close();
	    ob_end_flush();
	}
```